dnsmasq.sls

```diff
@@ -0,0 +1,32 @@
+# Use dnsmasq as a workaround forward for DNS requests to the internal IPv6 nameserver,
```

Comment:
I'd remove this comment as there might be other usecases, and this usecase is already outlined in the README.